src/test/java/com/google/sps/MealServletTest.java

```diff
@@ -165,6 +166,50 @@ public void getMealListTest() throws IOException, ServletException {
         assertEquals(expected, actual);
     }
 
+    // Get a list of objects Meal from datastore by search request "potato onion".
+    // (Check if meal that has both occurrences of searching keywords would be added only once).
+    // Expected result: a JSON String of list with one object MEAL_1.
+    @Test
+    public void getMealListWithTwoKeyWordsTest() throws IOException, ServletException {
+        DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+        ds.put(createMealEntity(MEAL_1));
+        ds.put(createMealEntity(MEAL_2));
+        
+        MealServlet servlet = new MealServlet();
+        MockHttpServletRequest request = new MockHttpServletRequest();
+        MockHttpServletResponse response = new MockHttpServletResponse();
+        request.setPathInfo("/query=potato onion");
+        servlet.doGet(request, response);
+        List<Meal> meals = new ArrayList<>();
```

Comment:
Put the empty line before `List<Meal>` and remove one before `Gson gson = new Gson();`.

That way the empty lines (together with the one earlier) split the test into 3 typical steps of a test: Arrange, Act, Assert. (This `List` is part of preparing the assertion, and is not related to the Act part that is the `doGet()` call.)

Please do that change to other tests too.